src/V3EmitCBase.cpp

```diff
@@ -82,6 +82,7 @@ string EmitCBaseVisitor::cFuncArgs(const AstCFunc* nodep) {
 
 void EmitCBaseVisitor::emitCFuncHeader(const AstCFunc* funcp, const AstNodeModule* modp,
                                        bool withScope) {
+    if (funcp->slow()) puts("VL_ATTR_COLD ");
```

Comment:
I suggest push moving this in a pre-patch as not directly related (I think).

Reply:
clang requires this appears on the definition, if a later declaration follows, which can now happen due to how the __ALL.cpp is laid out with the new output files, so again kind of related, but will push this separately as this actually might be a fix for a yet unknown issue.

Reply:
... if a later declaration follows *with the attribute attached* that is